crates/workspace/src/workspace.rs

```diff
@@ -305,13 +305,31 @@ pub fn init(app_state: Arc<AppState>, cx: &mut AppContext) {
 
             if let Some(app_state) = app_state.upgrade() {
```

Comment:
By the way, I'd like to know if this [action handler](https://github.com/zed-industries/zed/blob/09e7b481b8bed10deaca023d5ede81d738c102a9/crates/workspace/src/workspace.rs#L299) is dead code? I never saw it get called in my testing since there's already [a listener](https://github.com/zed-industries/zed/blob/f67e87057e4872353007be93fd238f68eecdf9e3/crates/workspace/src/workspace.rs#L1601) which is set in the root view of the workspace.

Reply:
It’s the backup for when no windows are open, which I think is only possible on macOS.